src/Home.js

```diff
@@ -3,9 +3,11 @@ import AreaChart from "./components/AreaChart";
 import { useEffect, useState } from "react";
 import * as d3 from "d3";
 import Plate from "./components/Plate";
+import ProgressBar from "./components/ProgressBar";
 
 const Home = () => {
-  const [data, setData] = useState([]);
+  const [AreaChartData, setAreaChartData] = useState([]);
+  const [ProgressChartData, setProgressChartData] = useState([]);
```

Comment:
We usually use `CapitalizedNaming` for classes and components. variables are usually `lowerCaseNamed`. This is not a breaking condition, as long as the team agrees. Here `areaChartData` and `progressChartData` would probably be better.

Reply:
Noted it Binal. Will work based on the naming conventions as mentioned.